app/src/main/java/org/mozilla/tv/firefox/webrender/cursor/CursorModel.kt

```diff
@@ -0,0 +1,352 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.webrender.cursor
+
+import android.annotation.SuppressLint
+import android.graphics.PointF
+import android.view.KeyEvent
+import android.view.MotionEvent
+import androidx.annotation.CheckResult
+import io.reactivex.Observable
+import io.reactivex.rxkotlin.Observables
+import io.reactivex.subjects.BehaviorSubject
+import io.reactivex.subjects.PublishSubject
+import org.mozilla.tv.firefox.ScreenControllerStateMachine
+import org.mozilla.tv.firefox.ScreenControllerStateMachine.ActiveScreen.WEB_RENDER
+import org.mozilla.tv.firefox.ext.isUriYouTubeTV
+import org.mozilla.tv.firefox.framework.FrameworkRepo
+import org.mozilla.tv.firefox.session.SessionRepo
+import org.mozilla.tv.firefox.utils.Direction
+import org.mozilla.tv.firefox.utils.DirectionHelper
+
+// Constants that can be tweaked to adjust cursor behavior
```

Comment:
nit: Constants we expect to be tweaked...

Just clarifying that you can change other things but we don't expect you to.